### ISSUE TITLE
Changes only in edit mode

### ISSUE THREAD

**Issue by ralfrupf1976** [nextcloud/collectives#1452]
**Describe the bug**
An edited page only shows the changes in edit mode. When I click on done, the status before the change is displayed. If I click on edit again it shows the edited version again

**To Reproduce**
1. Go to a page
2. Edit a page
3. Click on done
4. Changes are gone

**Expected behavior**
The changes you entered should be visible even after editing is complete

**Server details:**
- Collectives app version: 2.14.2
- Nextcloud version: 29.0.5
- PHP Version: 8.2.22
- Database: PostgreSQL 14.13

**Client details:**
 - OS: Manjaro
 - Browser: Firefox
 - Browser version: 129.0.1
 - Device: [e.g. iPhone13, desktop]

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by juliushaertl** [comment]
Likely a duplicate of #480

**Comment by mejo-** [comment]
Thanks for your report @ralfrupf1976. The problem already got reported in #1437 and fixed in #1450. I'll publish a bugfix release of Collectives soon.
